resource/js/concept-mappings.js

```diff
@@ -52,9 +52,13 @@ const conceptMappingsApp = Vue.createApp({
   },
   template: `
     <div v-load-concept-page="loadMappings">
-      <div v-if="hasMappings" class="main-content-section p-5">
-        <concept-mappings :mappings="mappings"></concept-mappings>
-      </div>
+      <div class="main-content-section p-5">
+        <template v-if="hasMappings">
+          <concept-mappings :mappings="mappings"></concept-mappings>
+        </template>
+        <template v-else>
+          <i class="fa-solid fa-spinner fa-spin-pulse"></i>
+        </div>
```

Comment:
The closing tag here should be `</template>`, right?

I get this error in the JS console due to this:

```
[Vue warn]: Template compilation error: Element is missing end tag.
5  |            <concept-mappings :mappings="mappings"></concept-mappings>
6  |          </template>
7  |          <template v-else>
   |          ^
8  |            <i class="fa-solid fa-spinner fa-spin-pulse"></i>
9  |          </div> 
  at <App>
  ```